src/zigFormat.ts

```diff
@@ -68,13 +68,13 @@ export class ZigRangeFormatProvider implements vscode.DocumentRangeFormattingEdi
 
 function zigFormat(document: vscode.TextDocument) {
     const config = vscode.workspace.getConfiguration('zig');
-    const formatCommand = <string[]>config.get('formatCommand');
+    const zigPath = config.get<string>('zigPath') || 'zig';
 
     const options = {
-        cmdArguments: [],
+        cmdArguments: ['fmt', '--stdin'],
         notFoundText: 'Install the zig stage2 compiler from https://github.com/ziglang/zig',
```

Comment:
perhaps you should change this to a message telling them to add zig to their path